data/initrd/theme.file_list

```diff
@@ -70,7 +70,7 @@ e echo "defaultrepo:	`default_repo`" >>linuxrc.config
 
 e echo "KexecReboot:    1" >>linuxrc.config
 
-e echo "PTOptions:	AutoUpgrade,productprofile,addon,XVideo,Screenmode,specialproduct,reboot_timeout" >>linuxrc.config
+e echo "PTOptions:	AutoUpgrade,productprofile,addon,XVideo,Screenmode,specialproduct,reboot_timeout,LIBSTORAGE_*,YAST_*" >>linuxrc.config
```

Comment:
Just a question out of curiosity. As far as I understand, these options are case-insensitive. Isn't it? If so, there is any reason to list some of them with CamelCase, others in lower case and others in uppercase?

Reply:
Yes, this is the 'canonical' spelling - the spelling that ends up in install.inf. And YaST does do an exact match.

For example `autoupgrade=1` ends up as `AutoUpgrade: 1` in install.inf.